src/brr.mli

```diff
@@ -3338,6 +3338,18 @@ module Window : sig
       {{:https://developer.mozilla.org/en-US/docs/Web/API/Window/scrollY}
       vertically scrolled} by. *)
 
+  val inner_width : t -> int
+  (** [inner_width w] is the
+      {{:https://developer.mozilla.org/en-US/docs/Web/API/Window/innerWidth}interior
+      height} of the window in pixels, including the width of the vertical
```

Comment:
Could you please try to clarify if these are CSS pixels and, if it is, specify it ? (It's not clear from the MDN page you will have to dig in the standards).

Reply:
As specified [here](https://www.w3.org/TR/cssom-view-1/#css-pixels):

> All coordinates and dimensions for the APIs defined in this specification are
> in CSS pixels, unless otherwise specified.

And nothing is specified in the [inner_width specification](https://www.w3.org/TR/cssom-view-1/#dom-window-innerwidth), so I believe it is CSS pixels.

But, just in case, I made a [small experiment](https://choum.net/panglesd/test_inner_width.html): zoom without changing the window size, and inner_width changes.

Reply:
Thanks you! Millions of `brr` web devs are going to be grateful to you for having researched that.